react/src/components/organisms/TabContainer/tab.js

```diff
@@ -51,7 +54,8 @@ Tab.propTypes = {
   // When true, the tab will be open by default when used with TabContainer.
   default: PropTypes.bool,
   // The text of the tab.
-  title: PropTypes.string.isRequired
+  title: PropTypes.string.isRequired,
+  handleClick: PropTypes.func
```

Comment:
can you add a comment above this /** A callback function from the on click action */